lib/stytch/b2b_magic_links.rb

```diff
@@ -44,6 +124,65 @@ def initialize(connection)
+      #
+      # == Parameters:
+      # organization_id::
+      #   Globally unique UUID that identifies a specific Organization. The `organization_id` is critical to perform operations on an Organization, so be sure to preserve this value.
+      #   The type of this field is `String`.
+      # email_address::
+      #   The email address of the Member.
+      #   The type of this field is `String`.
+      # login_redirect_url::
+      #   The URL that the Member clicks from the login Email Magic Link. This URL should be an endpoint in the backend server that
+      #   verifies the request by querying Stytch's authenticate endpoint and finishes the login. If this value is not passed, the default login
+      #   redirect URL that you set in your Dashboard is used. If you have not set a default login redirect URL, an error is returned.
+      #   The type of this field is nilable `String`.
+      # signup_redirect_url::
+      #   The URL the Member clicks from the signup Email Magic Link. This URL should be an endpoint in the backend server that verifies
+      #   the request by querying Stytch's authenticate endpoint and finishes the login. If this value is not passed, the default sign-up redirect URL
+      #   that you set in your Dashboard is used. If you have not set a default sign-up redirect URL, an error is returned.
+      #   The type of this field is nilable `String`.
+      # pkce_code_challenge::
+      #   A base64url encoded SHA256 hash of a one time secret used to validate that the request starts and ends on the same device.
+      #   The type of this field is nilable `String`.
+      # login_template_id::
+      #   Use a custom template for login emails. By default, it will use your default email template. The template must be from Stytch's
+      # built-in customizations or a custom HTML email for Magic Links - Login.
+      #   The type of this field is nilable `String`.
+      # signup_template_id::
+      #   Use a custom template for signup emails. By default, it will use your default email template. The template must be a template
+      #   using our built-in customizations or a custom HTML email for Magic Links - Signup.
+      #   The type of this field is nilable `String`.
+      # locale::
+      #   Used to determine which language to use when sending the user this delivery method. Parameter is a [IETF BCP 47 language tag](https://www.w3.org/International/articles/language-tags/), e.g. `"en"`.
+      #
+      # Currently supported languages are English (`"en"`), Spanish (`"es"`), and Brazilian Portuguese (`"pt-br"`); if no value is provided, the copy defaults to English.
+      #
+      # Request support for additional languages [here](https://docs.google.com/forms/d/e/1FAIpQLScZSpAu_m2AmLXRT3F3kap-s_mcV6UTBitYn6CdyWP0-o7YjQ/viewform?usp=sf_link")!
```

Comment:
there's zero percent chance this link still works in 3 years, but I think that's fine...